docs/azure-setup.md

```diff
@@ -79,7 +79,7 @@ An ARM template can be opened within Azure Portal via the following REST API:
 
 The URI can be one for either an Azure Blob or a GitHub file.  For example,
 
-    https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Ftongtest.blob.core.windows.net%2Fcntk%2FazureDeployMainTemplate.json
+    https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fmmlspark%2Fmaster%2Ftools%2Fdeployment%2FazureDeployMainTemplate.json
```

Comment:
It is not a good idea to depend on a path from github in this way.  I'd much rather see some solution that doesn't depend on GH at all.  (And I prefer to keep the `tongtest` since it's the magic string of all places that need to be updated when everything is sorted out.)